src/recipe/Baked.Recipe.Service.Application/DataAccess/DataAccessExtensions.cs

```diff
@@ -20,4 +22,10 @@ public static void ConfigureAutoPersistenceModel(this LayerConfigurator configur
 
     public static void ConfigureNHibernateInterceptor(this LayerConfigurator configurator, Action<InterceptorConfiguration> configuration) =>
         configurator.Configure(configuration);
+
+    public static void AClearSession(this Stubber giveMe)
+    {
+        giveMe.The<ISession>().Flush();
+        giveMe.The<ISession>().Clear();
+    }
```

Comment:
`GiveMe` stubber usually returns an object, this usage can be missleading

Reply:
i'll refactor this `GiveMe.TheSession(clear: true)` and update all `GiveMe.The<ISession>()` usages, how about this?

Reply:
That will be much better